serialization_test.go

```diff
@@ -64,6 +65,8 @@ func TestRecursiveJSON(t *testing.T) {
 		err := SendJSON(w, nil, value)
 
 		require.Error(t, err)
+		var unsupportedValueError *json.UnsupportedValueError
```

Comment:
Is this left over from #200? Not sure how it relates to this PR

Reply:
Yes sorry